packages/grafana-flamegraph/src/FlameGraphContainer.tsx

```diff
@@ -0,0 +1,252 @@
+import { css } from '@emotion/css';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useMeasure } from 'react-use';
+
+import { DataFrame, GrafanaTheme2 } from '@grafana/data';
+
+import FlameGraph from './FlameGraph/FlameGraph';
+import { FlameGraphDataContainer } from './FlameGraph/dataTransform';
+import FlameGraphHeader from './FlameGraphHeader';
+import FlameGraphTopTableContainer from './TopTable/FlameGraphTopTableContainer';
+import { MIN_WIDTH_TO_SHOW_BOTH_TOPTABLE_AND_FLAMEGRAPH } from './constants';
+import { ClickedItemData, ColorScheme, ColorSchemeDiff, SelectedView, TextAlign } from './types';
+
+type Props = {
```

Comment:
What do you think about also exporting this type? If someone for some reason would like to wrap the Graph in a HOC or similar it could be useful to be able to import the Props type.

Reply:
Ah yeah will do, definitely makes sense to export